octodns_ultra/__init__.py

```diff
@@ -371,6 +389,7 @@ def _apply(self, plan):
             self._post('/v2/zones', json=data)
             self.zones.append(name)
             self._zone_records[name] = {}
+            self._force_root_ns_update(changes)
```

Comment:
```suggestion
            changes = self._force_root_ns_update(changes)
```

Reply:
Minor tweak to just make it inherently obvious that `_force_root_ns_update` is going to be modifying changes. I'll go ahead and apply it to avoid yet another round trip.

Reply:
Actually looks like I don't have perms to commit to your branch. Will queue up the same change over on the ns1 version.